### ISSUE TITLE
Release Auto Slippage model to reduce fail rate <10%

### ISSUE THREAD

**Issue by asood123** [Uniswap/interface#1495]
Per data discussion in team meeting

**Comment by asood123** [comment]
1. Get data for Add Liquidity
* if high, River can run data

2. Get data for Swaps from last 24 hours (sampled from Alchemy)

3. Move forward with tenderly to get this data by early to mid next week
* Aseem/River to list out exact breakdown for tenderly
* figure out the data we want: split by swap/add, specific pools, error codes, etc

4. Explore signals for Auto (Will/Noah to file a separate issue with a proposal)
* based on dollar values, fee tier, stablecoin pairs, etc
* Ex: fixed slippage + additional factors based on signals

**Comment by willhennessy** [comment]
* Fail rate is down to 17.5%
* Line of sight to a data-driven auto slippage model in the next 2 weeks with data from tenderly and block native
* Decided to not pursue heuristic auto slippage in the interim. Will stay at 0.5% for now.

**Comment by willhennessy** [comment]
* blocked on data delivery from block native and tenderly. ETA: 1-2 weeks.
* then few weeks to implement the model 

fail rate is at 19% today

**Comment by willhennessy** [comment]
blocked on issues #1846 #1847 #1851 #1848 in progress this week
